assets/helpers/abtestDefinitions.js

```diff
@@ -50,4 +50,28 @@ export const tests: Tests = {
     isActive: true,
     independence: 3,
   },
+
+  ukRecurringAmountsTest: {
+    variants: ['control', 'higher', 'lower', 'wildcard'],
+    audiences: {
+      GB: {
+        offset: 0,
+        size: 1,
+      },
+    },
+    isActive: true,
+    independence: 4,
+  },
+
+  usRecurringAmountsTest: {
+    variants: ['control', 'higher', 'lower', 'wildcard'],
+    audiences: {
+      US: {
```

Comment:
Under the current ABtest framework, it is possible to implement a test with different segments in different countries. In that way, this will be implemented as one test and it will simplify the validation in  `contribAmounts.jsx`